modules/user/server-ts/social/google/index.js

```diff
@@ -1,11 +1,12 @@
 import { pick } from 'lodash';
 import passport from 'passport';
 import { OAuth2Strategy as GoogleStrategy } from 'passport-google-oauth';
+import { access } from '@module/authentication-server-ts';
```

Comment:
Some middlewares should be located in `auth` module. These files are needed to be split as well.

Reply:
Done.